ui/types/vault/utils/client-count-utils.d.ts

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/*
+The client count utils are responsible for serializing the sys/internal/counters/activity API response
+The initial API response shape (line ~82) and serialized types are defined here and used to
+defines the activity model in models/clients/activity.d.ts
+
+To help visualize the response there are sample responses in ui/tests/helpers/clients.js
+*/
+
+// TYPES RETURNED BY UTILS (serialized)
+
+export interface TotalClients {
```

Comment:
I would export these from the util itself instead, because the `.d.ts` are supposed to be placeholder type files for when you don't have a file written in typescript. Or, I think we can export a regular `.ts` file from this folder with just the types if you want to keep the types separete from the util file

Reply:
Okay great! I meant to leave a comment here asking where these should go. I only moved them here because they're pretty long. But that answers my question! Thank you